pages/index.vue

```diff
@@ -65,15 +65,30 @@ import ContentData from '@/static/content.json'
 */
 const importAll = (req, next) => {
   const files = req.keys()
-  if (files.length > 0) { return next(req.keys()) }
+
+  files.sort(function (a, b) {
+    if (a === b) {
+      return 0
+    }
+
+    if (a === './mainnet.json' || (a === './testnet.json' && b !== './mainnet.json')) {
+      return -1
+    }
+
+    if (b === './mainnet.json' || b === './testnet.json') {
+      return 1
+    }
+    return a.localeCompare(b)
+  })
+  if (files.length > 0) { return next(files) }
```

Comment:
The idea here is that mainnet always appears first in the list and testnet second in the list, the rest should appear alphabetically. Not super clean, happy to take a cleaner solution if exists.